velox/vector/fuzzer/tests/VectorFuzzerTest.cpp

```diff
@@ -173,6 +174,60 @@ TEST_F(VectorFuzzerTest, flatNotNull) {
   ASSERT_FALSE(vector->mayHaveNulls());
 }
 
+struct Foo {
+  explicit Foo(int64_t id) : id_(id) {}
+  int64_t id_;
+};
+
+TEST_F(VectorFuzzerTest, flatOpaque) {
+  // Exercises fuzzer.flatOpaque()
+  VectorFuzzer::Options opts;
+  opts.nullRatio = 0.5;
+  VectorFuzzer fuzzer(opts, pool());
+  fuzzer.registerOpaqueTypeGenerator<Foo>([](FuzzerGenerator& rng) {
```

Comment:
Could you register two OpaqueTypeGenerators for Foo and Bar to ensure the correct one can be retrieved?